pkg/asset/machines/master.go

```diff
@@ -402,6 +402,14 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 		}
 		machineConfigs = append(machineConfigs, ignFIPS)
 	}
+	if len(ic.Workload) > 0 {
+		w := ic.Workload[0]
+		ignWorkload, err := machineconfig.ForWorkloadPartitioning(string(w.Name), w.CPUIds, "master")
+		if err != nil {
+			return errors.Wrap(err, "failed to create ignition for workload partitioning for master machines")
+		}
+		machineConfigs = append(machineConfigs, ignWorkload)
+	}
```

Comment:
Do this for all workload partitions now so that we don't have to change this later when we support more.
```suggestion
	for _, w := range ic.Workload {
		ignWorkload, err := machineconfig.ForWorkloadPartitioning(string(w.Name), w.CPUIds, "master")
		if err != nil {
			return errors.Wrap(err, "failed to create ignition for workload partitioning for master machines")
		}
		machineConfigs = append(machineConfigs, ignWorkload)
	}
```

Reply:
Can do.

Reply:
Turns out it works better to combine all of the partitions into a single machineconfig, since bot the CRI-O and Kubelet config file drop-ins are designed to handle multiple partitions in each, so I did it that way.  Push with the changes coming soon.